usage.md

```diff
@@ -66,6 +67,86 @@ Returns the Smart Address as per ERC-4337, the SCW address, associated with the
 
 Returns the balance available in the gas tank that can be utilized via the current user's SCW address.
 
+### `createSession()`
+
+Creates session keys using the [`CreateSessionParam`](https://gasless-sdk-ref-guide.netlify.app/types/createsessionparam) policy and requests the user for permission. When the user initiates any app operation that fulfills the session policy, it will be autonomously approved.
+
+```ts
+  import { SCW, StorageType  } from "@arcana/scw";
+
+  let scWallet: SCW;  
+
+  type CreateSessionParam = {
+    contractAddress: string;
+    functionSelector: string;
+    rules: string[];
```

Comment:
It a array of `Rule` object (imported from biconomy) not string

Reply:
fixed.